### PR TITLE
10778: V2 XML Issues-POC and office locations

### PR DESCRIPTION
**Pull request by sagarlekar**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [10778](https://devops.dchbx.org/redmine/issues/10778) |
| App-Dev Road Map # | n/a |
| Start Date | 09/12/2016 |
| Due Date | n/a  |
| App-Dev Priority | UP-3 |
| Development Story Points | n/a |
| Peer Review Story Points | n/a |
| Ticket Author | @LavinaNicole |

### CODE REVIEW

File: app/views/events/v2/phones/_phone.xml.haml [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-    - if !phone.extension.blank?
-      %extension= phone.extension
-    %is_preferred= (phone.primary.blank? ? false : phone.primary)
+%phone
+  %type= "urn:openhbx:terms:v1:phone_type##{phone.kind}"
+  - if !phone.country_code.blank?
+    %country_code=phone.country_code
+  %area_code=phone.area_code
+  %phone_number=phone.number
+  %full_phone_number= phone.area_code.to_s + phone.number
```
Comment (TreyE) [inline app/views/events/v2/phones/_phone.xml.haml]: Why doesn't this use the full phone number accessor?

Comment (rabbittracey) [inline app/views/events/v2/phones/_phone.xml.haml]: because the according xml file in enroll generates the full phone number in this format

Comment (sagarlekar) [inline app/views/events/v2/phones/_phone.xml.haml]: There were cases reported by Joe where phone.full_phone_number was empty but phone.area_code and phone.number were populated hence using these instead

Comment (sagarlekar) [inline app/views/events/v2/phones/_phone.xml.haml]: @saimekala07 has fixed the empty full phone number issue in https://devops.dchbx.org/redmine/issues/7967
I have changed the code to use `%full_phone_number= phone.full_phone_number`

Comment (saimekala07) [inline app/views/events/v2/phones/_phone.xml.haml]: Before we had issue with full phone number this PR https://github.com/dchbx/enroll/pull/163/files fixed full phone number attribute.Now we can directly access full phone number.

**Pull request merged**